.github/workflows/deepfactor.yml

```diff
@@ -0,0 +1,16 @@
+name: deepfactor-scan
+on:
+  push:
+    branches: [ main, develop, feature/deepfactor ]
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    container:
+      image: public.ecr.aws/deepfactor/df/runtime:3.3.3-2346
```

Comment:
I wonder if we will also want to make the df runtime version number an org-wide variable, so that we don't have to manually update too many instance of it when the time comes to update/upgrade.

Reply:
That sounds like a good idea, @forevermatt, but I think that's something that only you or @devon-sil can do.